src/resources/api/announcement/DeleteAnnouncements.js

```diff
@@ -0,0 +1,23 @@
+import api from "../config";
+
+let apiPrefix = "/badaso-api";
+
+export const DeleteAnnouncement = async (id) => {
```

Comment:
please use camelCase for function name. And since you declare this in a file named the same with the function name, I guess `export default` is more preferred.

Even better, I think all the `createAnnouncement`, `editAnnouncement`, `deleteAnnouncement` etc can be put into one single file `api/announcement.js`. It's neater imho

Reply:
I prefer the second option